release_notes/unreleased.md

```diff
@@ -1 +1,5 @@
 **Unreleased**
+* Bug fixed in 'update hash' action
```

Comment:
Can we be more specific about which bug fix is done here?

Reply:
Updated